_docs/_latest/configure/scanner/rules.md

```diff
@@ -215,14 +260,35 @@ rules:
           - YOUR_ORG_ID / YOUR_PROJECT_ID
 ```
 
- - **name**: The description of your rule.
- - **bucket**: The bucket name you want to audit.
- - **entity**: The [ACL entity](https://cloud.google.com/storage/docs/access-control/lists)
-   that holds the bucket permissions.
- - **email**: The email of the entity.
- - **domain**: The domain of the entity.
- - **role**: The access permission of the entity.
- - **resource**: The resource under which the bucket resides.
+- `name`
+  * **Description**: The name of the rule.
+  * **Valid values**: String.
+
+ - `resource`
+    - `resource_ids`
+      * **Description**: A list of one or more resource ids to match.
+      * **Valid values**: String, you can use `*` to match for all.
+
+- `bucket`
+  * **Description**: The bucket name you want to audit.
+  * **Valid values**: String, you can use `*` to match for all.
+
+- `entity`
+  * **Description**: The [ACL entity](https://cloud.google.com/storage/docs/access-control/lists) that holds the bucket permissions.
+  * **Valid values**: String.
+  * **Example values**: `AllUsers`
+
+- `email`
+  * **Description**: The email of the entity.
+  * **Valid values**: String, you can use `*` to match for all.
+
+- `domain`
+  * **Description**: The domain of the entity.
+  * **Valid values**: String, you can use `*` to match for all.
+
+- `role`
```

Comment:
nit: You've mixed your unordered lists demarks, `-` and `*`. Fix throughout.

Reply:
Updated.